packages/components/src/components/as-category-widget/as-category-widget.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../styles/src/core/_dev';
+
+$bar--height: get-spacing(1);
+
+as-category-widget {
+  display: block;
+
+  @include padding(4);
+
+  :root {
+    --description--color: $color-type-20;
```

Comment:
why :root, why not declaring the css-vars under the `as-category-widget` namespace

Reply:
Yes, it would be better doing it that way.